src/pint/utils.py

```diff
@@ -1667,3 +1673,90 @@ def remove_dummy_distance(c):
             "Do not know coordinate frame for %r: returning coordinates unchanged" % c
         )
         return c
+
+
+def info_string(prefix_string="# ", comment=None):
+    """
+    Returns an informative string about the current state of PINT.
+
+    Adds:
```

Comment:
Does this render okay in the docs? `make docs` may work if you have pandoc, or if you resolve the merge conflict and CI runs you can use the link in the readthedocs job to see what this docstring looks like. (Maybe it won't show up - is this function in `__all__`?)

Reply:
Working on this.  Is there an easy way to run `make docs` just on 1 file (the whole build takes a while).

Reply:
Yeah I know it's unpleasant. `(cd docs && make html)` is the best I've got - it doesn't clear out intermediate Sphinx files so it's a little faster but it still takes a while.